OpenRA.Mods.D2k/Traits/AttractsWorms.cs

```diff
@@ -0,0 +1,36 @@
+#region Copyright & License Information
+/*
+* Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+* This file is part of OpenRA, which is free software. It is made
+* available to you under the terms of the GNU General Public License
+* as published by the Free Software Foundation. For more information,
+* see COPYING.
+*/
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.D2k.Traits
+{
+	[Desc("This actor makes noise, which causes them to be targeted by actors with the Sandworm trait.")]
+	public class AttractsWormsInfo : ITraitInfo
+	{
+		[Desc("How much noise this actor produces.")]
+		public readonly int Intensity = 0;
+
+		[Desc("How much does the sound subside with each cell (ammount of intensity units lost per cell).")]
```

Comment:
This is a question that is incorrectly ended with a `.`. Change this to a `?` or make this a statement.